brayns/engine/model/ModelInstance.cpp

```diff
@@ -75,19 +74,8 @@ const Bounds &ModelInstance::getBounds() const noexcept
 
 void ModelInstance::computeBounds() noexcept
 {
-    Log::debug("[ModelInstance {}] Computing bounds", _modelInstanceID);
-
-    const auto matrix = _transform.toMatrix();
-    _bounds = _model.computeBounds(matrix);
-
-    // In the event that the model used by this instance has no geometry,
-    // The result of computeBounds() would be bounds with min > max.
-    const auto &min = _bounds.getMin();
-    const auto &max = _bounds.getMax();
-    if (glm::max(min, max) == min)
-    {
-        _bounds = Bounds(Vector3f(0.f), Vector3f(0.f));
-    }
+    Log::debug("[ModelInstance {}] Computing bounds", _instanceID);
```

Comment:
I don't think it is the responsibility of the model instance to log stuff but more for the caller at a higher level but not a big deal...

Reply:
I agree. Furthermore this was used just to debug the first refactor and it is not needed anymore.